### PR TITLE
Add limit to paginator

### PR DESCRIPTION
**Pull request by codealchemy**
Addresses issue [#11](https://github.com/nationbuilder/nationbuilder-rb/issues/11), adds support for `limit` param when paginating through requests.

cc: @kaip @ccollins

### CODE REVIEW

File: lib/nationbuilder/paginator.rb [hunk 1]
```diff
@@ -11,10 +11,11 @@ def initialize(client, body)
       @body[page_type.to_s]
     end
 
-    define_method(:"#{page_type}") do
+    define_method(:"#{page_type}") do |args = {}|
       return nil unless send(:"#{page_type}?")
       path = send(:"#{page_type}?").split('/api/v1').last
-      results = @client.raw_call(path, :get)
+      args[:limit] ||= CGI.parse(path)['limit']
+      results = @client.raw_call(path, :get, args)
```
Comment (kaip) [inline lib/nationbuilder/paginator.rb]: Why do we pass the args to the client raw_call now?

Comment (codealchemy) [inline lib/nationbuilder/paginator.rb]: To allow passing in of the `limit` param, it [defaulted](https://github.com/nationbuilder/nationbuilder-rb/blob/master/lib/nationbuilder/client.rb#L42) to an empty hash anyways when it wasn't included in the pagination before. Looking at this again, I'll update this to read `body` to be more in line with how the raw call is written.

**Comment by kaip** [comment]
:+1:

**Comment by codealchemy** [comment]
@ccollins

**Comment by ccollins** [comment]
:sparkles: :sparkling_heart: :sparkles:

**Pull request merged**